release/build_release_package.sh

```diff
@@ -12,7 +12,7 @@ xmlstarlet edit -L -S -O -u "/_/version" -v "${TRAVIS_TAG}" src/var/connect/shop
 xmlstarlet edit -L -S -u "/config/modules/Shopgate_Cloudapi/version" -v "${TRAVIS_TAG}" src/app/code/community/Shopgate/Cloudapi/etc/config.xml
 
 # Install magento to create package
-wget --quiet -O release/magento1702.zip http://files.shopgate.com/magento/magento1702.zip > /dev/null
+#wget --quiet -O release/magento1702.zip http://files.shopgate.com/magento/magento1702.zip > /dev/null
```

Comment:
This is not going to work across the board, you have to create an if statement if you want it to work this way.
 E.g. check if the zip exists, if not, download it.

Reply:
This is my local stash. I did not mean to push it!